NICE.Identity.Authorisation.WebAPI/Repositories/IdentityContext.cs

```diff
@@ -48,14 +48,14 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     .IsRequired()
                     .HasMaxLength(50);
 
-                entity.HasData(new Environment[] {
-                    new Environment(1, "Local", 10),
-                    new Environment(2, "Dev", 20),
-                    new Environment(3, "Test", 30),
-                    new Environment(4, "Alpha", 40),
-                    new Environment(5, "Beta", 50),
-                    new Environment(6, "Live", 60),
-                });
+                //entity.HasData(new Environment[] {
```

Comment:
Are we keeping this here but commented out for a reason?

Reply:
no, i've removed it now.

the bit confusing me was that that code doesn't appear to have ever worked either. the order was 0 on alpha. 

it might be the reserved word thing i was on about, or it might be a sql version issue specific to alpha. no idea, but at least the SQL approach seems to have worked fine.